### PR TITLE
After removing RegisterMap() application doesn't work properly #17

### PR DESCRIPTION
**Pull request by aloscha**
After this pr, Chatter also should works with new version of ImageApp

### CODE REVIEW

**Comment by warpech** [comment]
In future, could you please commit refactorings as a separate commit? That makes code review easier.

**Comment by warpech** [comment]
For me it is hard to understand what I am reviewing. How is the introduction of `PreviewObject` related to the problem `After removing RegisterMap() application doesn't work properly`?

[Here is a great short article](http://rickardlindberg.me/writing/problem-in-commit-message/) that proposes how to write commit messages according to a template:

```
Problem: What is the problem?

Solution: How does the change in this commit fix the problem?
```

**Comment by warpech** [comment]
ping @aloscha. Can you explain what is the purpose of changes in this PR?

**Comment by warpech** [comment]
ping @aloscha

**Comment by aloscha** [comment]
>After removing RegisterMap() application doesn't work properly

Problem: After I remove RegisterMap() from Code - So application won't support mapping application shouldn't crash - https://github.com/StarcounterApps/Chatter/blob/master/Api/MainHandlers.cs#L89

**Comment by warpech** [comment]
I was expecting something like:

Problem: The app relies on UriMapping to display its own partial to create a new message.

File: Api/MainHandlers.cs [hunk 1]
```diff
@@ -128,7 +116,8 @@ internal class MainHandlers {
                 var page = new ChatMessagePage
                 {
                     Data = message,
-                    PreviewText = Self.GET("/chatter/partials/chatattachmenttextpreview/" + objectId)
+                    PreviewText = Self.GET("/chatter/partials/chatattachmenttextpreview/" + objectId),
+                    PreviewObject = Self.GET("/chatter/partials/chatattachmentobjectpreview/" + objectId),
```
Comment (warpech) [inline Api/MainHandlers.cs]: How was adding a new partial `PreviewObject` as solution to the problem? Would it not be possible with just `PreviewText`?

**Comment by warpech** [comment]
Closing this as I think this is not an acceptable solution, because:

1. I think #17 is actually not an issue
2. If #17 is a confirmed problem, then the solution to it should not add more complexity to the app

**Pull request closed without merging**